purecharity-wp-givingcircles/purecharity-wp-givingcircles.php

```diff
@@ -90,4 +90,32 @@ function gc_force_template() {
 	$options = get_option( 'purecharity_givingcircles_settings' );
   include(TEMPLATEPATH . '/' . $options['single_view_template']);
   exit;
+}
+
+/*
+ * Plugin updater using GitHub
+ *
+ * Auto Updates through GitHub
+ *
+ * @since   1.1.2
+ */
+add_action( 'init', 'purecharity_wp_givingcircles_updater' );
+function purecharity_wp_givingcircles_updater() {
+  if ( is_admin() ) { 
+    $gc_config = array(
+      'slug' => plugin_basename( __FILE__ ),
+      'proper_folder_name' => 'purecharity-wp-givingcircles',
+      'api_url' => 'https://api.github.com/repos/purecharity/pure-givingcircles',
+      'raw_url' => 'https://raw.github.com/purecharity/pure-givingcircles/master',
+      'raw_url' => 'https://raw.githubusercontent.com/purecharity/pure-givingcircles/master/purecharity-wp-givingcircles/',
```

Comment:
I'm not sure if this is intentional but there are two entries for `raw_url` here